.github/workflows/build_and_test.yaml

```diff
@@ -0,0 +1,61 @@
+See also https://github.com/EVerest/everest-core/tree/main/.github/workflows
+# TODO: modify to reuse the above workflow to DRY up CI.
+
+name: Build and test libtimer
+on: 
+  pull_request: {}
+  push:
```

Comment:
```suggestion
```

This trick seems to not work for external branches. In this case I would suggest to merge this PR after approving and after this testing on main branch. If needed we can do a second fixing PR then..

Since branch protection rules don't need to include this check instantly it won't brake or block anything

Reply:
Even better would be if you could provide a link to a successfull run in of this workflow in your fork

Reply:
https://github.com/US-JOET/libtimer/actions/runs/7821295738 - passing within our forked repo.